examples/ex21p_proposed.cpp

```diff
@@ -0,0 +1,613 @@
+//                       MFEM Example 21 - Parallel Version
```

Comment:
I think this is a good example, lets just rename it to ex21.

Any chance for a serial version?

Reply:
I'd be happy to put that together.  It will be interesting to see how the solvers perform without preconditioning.